### ISSUE TITLE
Low accuracy when pretraining on synthetic dataset

### ISSUE THREAD

**Issue by davidpengiupui** [JitengMu/Learning-from-Synthetic-Animals#12]
Hi!

Thank you so much for your work. I tried to reproduce it but obtained really low accuracies for either syn or ssl. I used the same settings as yours. Here is the log from /log/horse/syn. Could you please give some hints on how to solve it? Thank you.

Epoch	LR	Train Loss	Val Loss	Train Acc	Val Acc	Val Auc	Val Mean Error	
1.000000	0.000250	0.002964	0.001544	0.023988	0.072483	5.500000	2.194995	
2.000000	0.000250	0.001965	0.001527	0.058656	0.096264	7.150000	2.367952	
3.000000	0.000250	0.001946	0.001929	0.070092	0.101267	7.430000	2.253990	
4.000000	0.000250	0.001939	0.001530	0.081494	0.111642	8.250000	2.364837	
5.000000	0.000250	0.001934	0.001540	0.091778	0.109156	8.070000	2.321525	
6.000000	0.000250	0.001912	0.003664	0.099414	0.040644	3.030000	4.187365	
7.000000	0.000250	0.001903	0.179951	0.113940	0.017394	1.310000	4.251516	
8.000000	0.000250	0.001834	0.001524	0.121116	0.082683	6.120000	2.742665	
9.000000	0.000250	0.001629	0.001545	0.121600	0.106158	7.680000	2.738890	
10.000000	0.000250	0.001502	0.001504	0.150396	0.112214	8.270000	2.331060	
11.000000	0.000250	0.001479	0.001530	0.162228	0.117617	8.550000	2.313860	
12.000000	0.000250	0.001478	0.001500	0.169594	0.112178	8.270000	2.224269	
13.000000	0.000250	0.001466	0.001498	0.180773	0.114069	8.330000	2.273391	
14.000000	0.000250	0.001460	0.001496	0.187913	0.110219	8.100000	2.243525	
15.000000	0.000250	0.001456	0.001495	0.196187	0.107961	7.970000	2.185828	
16.000000	0.000250	0.001451	0.001493	0.200994	0.118764	8.670000	2.356410	
17.000000	0.000250	0.001444	0.001494	0.209292	0.120528	8.880000	2.357964	
18.000000	0.000250	0.001439	0.001493	0.219391	0.116286	8.550000	2.325340	
19.000000	0.000250	0.001436	0.001492	0.223222	0.119786	8.770000	2.310177	
20.000000	0.000250	0.001430	0.001492	0.226854	0.120422	8.760000	2.331233	
21.000000	0.000250	0.001428	0.001493	0.233863	0.121603	8.800000	2.373088	
22.000000	0.000250	0.001422	0.001492	0.241863	0.118264	8.580000	2.369488	
23.000000	0.000250	0.001420	0.001492	0.240275	0.121817	8.810000	2.316702	
24.000000	0.000250	0.001415	0.001492	0.247376	0.116522	8.470000	2.392433	
25.000000	0.000250	0.001408	0.001492	0.253778	0.115944	8.490000	2.400347	
26.000000	0.000250	0.001406	0.001492	0.257598	0.117103	8.520000	2.358543	
27.000000	0.000250	0.001402	0.001492	0.261179	0.119886	8.700000	2.375059	
28.000000	0.000250	0.001401	0.001492	0.264578	0.111553	8.190000	2.308105	
29.000000	0.000250	0.001398	0.001492	0.268355	0.107556	7.910000	2.372019	
30.000000	0.000250	0.001393	0.001492	0.270940	0.117625	8.630000	2.366384	
31.000000	0.000250	0.001392	0.001492	0.271940	0.115508	8.500000	2.391092	
32.000000	0.000250	0.001386	0.001492	0.278610	0.118797	8.710000	2.355343	
33.000000	0.000250	0.001385	0.001492	0.278522	0.112708	8.360000	2.310559	
34.000000	0.000250	0.001382	0.001492	0.282694	0.120169	8.770000	2.369975	
35.000000	0.000250	0.001381	0.001494	0.284552	0.100664	7.300000	2.190351	
36.000000	0.000250	0.001376	0.001492	0.288251	0.119811	8.790000	2.372497	
37.000000	0.000250	0.001373	0.001493	0.289277	0.120472	8.770000	2.311811	
38.000000	0.000250	0.001372	0.001493	0.293529	0.105972	7.780000	2.355505	
39.000000	0.000250	0.001372	0.001492	0.291821	0.120206	8.800000	2.374719	
40.000000	0.000250	0.001365	0.001492	0.300266	0.119539	8.760000	2.360193	
41.000000	0.000250	0.001367	0.001491	0.297209	0.117389	8.630000	2.382852	
42.000000	0.000250	0.001363	0.001492	0.302322	0.123283	9.040000	2.345582	
43.000000	0.000250	0.001359	0.001492	0.304242	0.119011	8.720000	2.320885	
44.000000	0.000250	0.001358	0.001491	0.306355	0.123792	9.040000	2.351524	
45.000000	0.000250	0.001358	0.001496	0.307076	0.092775	6.890000	2.168275	
46.000000	0.000250	0.001358	0.001494	0.307551	0.113064	8.260000	2.362809	
47.000000	0.000250	0.001355	0.001492	0.306246	0.121297	8.910000	2.346479	
48.000000	0.000250	0.001353	0.001492	0.308777	0.121536	8.940000	2.384686	
49.000000	0.000250	0.001350	0.001492	0.312204	0.122933	9.030000	2.359059	
50.000000	0.000250	0.001351	0.001492	0.310291	0.125275	9.100000	2.345671	
51.000000	0.000250	0.001346	0.001492	0.319651	0.123225	8.920000	2.403006	
52.000000	0.000250	0.001345	0.001491	0.317297	0.124411	9.030000	2.349514	
53.000000	0.000250	0.001342	0.001491	0.320758	0.121619	8.940000	2.365828	
54.000000	0.000250	0.001343	0.001492	0.319759	0.120903	8.890000	2.312042	
55.000000	0.000250	0.001339	0.001495	0.325148	0.108606	7.960000	2.341037	
56.000000	0.000250	0.001336	0.001491	0.326207	0.120219	8.800000	2.375302	
57.000000	0.000250	0.001337	0.001491	0.323702	0.120328	8.870000	2.376870	
58.000000	0.000250	0.001337	0.001491	0.326113	0.124317	9.070000	2.340797	
59.000000	0.000250	0.001336	0.001493	0.325525	0.105014	7.760000	2.302611	
60.000000	0.000250	0.001337	0.001492	0.325296	0.115703	8.440000	2.379253	
61.000000	0.000025	0.001324	0.001490	0.334911	0.122475	8.940000	2.357625	
62.000000	0.000025	0.001313	0.001491	0.342325	0.123267	9.020000	2.381993	
63.000000	0.000025	0.001314	0.001491	0.344097	0.123794	9.020000	2.375047	
64.000000	0.000025	0.001312	0.001490	0.347682	0.123983	9.030000	2.362030	
65.000000	0.000025	0.001310	0.001491	0.347910	0.123169	8.980000	2.389049	
66.000000	0.000025	0.001309	0.001490	0.345883	0.123850	9.050000	2.368623	
67.000000	0.000025	0.001304	0.001491	0.352021	0.124089	9.000000	2.377154	
68.000000	0.000025	0.001314	0.001491	0.348625	0.125089	9.120000	2.359371	
69.000000	0.000025	0.001303	0.001491	0.350167	0.124683	9.130000	2.358260	
70.000000	0.000025	0.001305	0.001490	0.352417	0.124150	9.090000	2.356330	
71.000000	0.000025	0.001303	0.001490	0.352169	0.125211	9.120000	2.357595	
72.000000	0.000025	0.001303	0.001491	0.350169	0.122006	8.930000	2.372578	
73.000000	0.000025	0.001302	0.001491	0.354385	0.124472	9.080000	2.362327	
74.000000	0.000025	0.001305	0.001490	0.351102	0.124817	9.120000	2.362581	
75.000000	0.000025	0.001303	0.001490	0.353623	0.125178	9.130000	2.367403	
76.000000	0.000025	0.001300	0.001491	0.353802	0.125167	9.130000	2.354573	
77.000000	0.000025	0.001302	0.001491	0.355727	0.123975	9.070000	2.359106	
78.000000	0.000025	0.001300	0.001490	0.354267	0.124903	9.130000	2.354754	
79.000000	0.000025	0.001301	0.001491	0.352326	0.124842	9.100000	2.354053	
80.000000	0.000025	0.001302	0.001490	0.353586	0.124431	9.100000	2.345293	
81.000000	0.000025	0.001296	0.001491	0.359643	0.124492	9.080000	2.364657	
82.000000	0.000025	0.001302	0.001491	0.354609	0.123514	9.030000	2.370108	
83.000000	0.000025	0.001299	0.001491	0.359552	0.123383	8.990000	2.373100	
84.000000	0.000025	0.001298	0.001490	0.357012	0.125608	9.180000	2.348228	
85.000000	0.000025	0.001300	0.001490	0.353865	0.124719	9.150000	2.348849	
86.000000	0.000025	0.001295	0.001491	0.362012	0.124872	9.090000	2.359938	
87.000000	0.000025	0.001296	0.001490	0.358111	0.124164	9.070000	2.362884	
88.000000	0.000025	0.001295	0.001491	0.361090	0.123867	9.060000	2.358939	
89.000000	0.000025	0.001293	0.001491	0.360145	0.124300	9.100000	2.369569	
90.000000	0.000025	0.001295	0.001490	0.362428	0.124308	9.090000	2.366930	
91.000000	0.000003	0.001295	0.001490	0.358858	0.124992	9.110000	2.359978	
92.000000	0.000003	0.001292	0.001490	0.360730	0.122683	8.990000	2.371102	
93.000000	0.000003	0.001294	0.001490	0.359592	0.123172	9.010000	2.372394	
94.000000	0.000003	0.001292	0.001490	0.359736	0.123267	8.980000	2.367979	
95.000000	0.000003	0.001293	0.001491	0.362945	0.123672	9.030000	2.374837	
96.000000	0.000003	0.001292	0.001490	0.362015	0.123256	9.010000	2.372088	
97.000000	0.000003	0.001294	0.001491	0.362357	0.122289	8.930000	2.368043	
98.000000	0.000003	0.001288	0.001490	0.367519	0.122394	8.970000	2.367084	
99.000000	0.000003	0.001292	0.001490	0.366913	0.124228	9.070000	2.373930	
100.000000	0.000003	0.001293	0.001490	0.359710	0.125639	9.150000	2.355735

**Comment by JitengMu** [comment]
I am not sure exactly why this is the case if you follow the instructions. Have you tried with the provided checkpoints to see the accuracy is still low? How about results on other categories?
